week-5/ghostdb/Scripts/2_script_data_temporal.sql

```diff
@@ -0,0 +1,999 @@
+SET datestyle = "ISO, MDY";
+INSERT INTO temporal VALUES ('1/2/09 6:17','Caribbean Tour',9,1200,'Mastercard','Holiday dreams','Jamie Lopez','carolina','Basildon','England','United Kingdom','Whatsapp');
```

Comment:
What does this file do?

Reply:
Setup the date style. It add ISO standard and MDY (month, day and year) regardless the OS you use.